libraries/llvm/rts.ll

```diff
@@ -291,7 +292,7 @@ define %StackPointer @stackDeallocate(%Stack %stack, i64 %n) {
 ; Meta-stack management
 
 define %Memory @newMemory() {
-    %stackPointer = call %StackPointer @malloc(i64 268435456)
+    %stackPointer = call %StackPointer @calloc(i64 268435456, i64 1)
```

Comment:
We should really grow the stack dynamically, otherwise initializing with 0 will decrease performance a lot. I added it here since Valgrind warned about uninitialized memory.